src/main/webapp/scripts/script.js

```diff
@@ -12,17 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 /**
- * Adds a random greeting to the page.
+ * Function to load the search vendors around bar
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+$(function(){
+  $("#searchBarVendors-placeholder").load("COMMONS/searchBarVendors.html"); 
```

Comment:
Nit: In JavaScript, we usually use single quotes. Also, directory names are usually lowercase.

```suggestion
  $('#searchBarVendors-placeholder').load('common/searchBarVendors.html'); 
```